src/main/java/racingcar/PrintView.java

```diff
@@ -0,0 +1,23 @@
+package racingcar;
+
+public class PrintView {
+	private static final String RESULT_MESSAGE = "실행 결과";
+
+	public static void printEmptyLine() {
+		System.out.println();
+	}
+
+	public static void printResultMessage() {
+		System.out.println("\n" + RESULT_MESSAGE);
+	}
+
+	public static void printCarMoving(boolean[] moving, int trial) {
```

Comment:
```suggestion
	public static void printCar(Car car) {
```
값을 꺼내서 view에게 전달하지 않고 객체 자체를 넘기면 좋겠습니다.